ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDbFactory.java

```diff
@@ -71,6 +71,9 @@ public HikariDataSource buildDataSource(DbEndpoint dbEndpoint) {
     hikariConfig.addDataSourceProperty("prepStmtCacheSize", "250");
     hikariConfig.addDataSourceProperty("prepStmtCacheSqlLimit", "2048");
     hikariConfig.addDataSourceProperty("useServerPrepStmts", "true");
+    if (!config.transactionIsolationLevel.equals(TransactionIsolationLevel.TRANSACTION_NONE)) {
+      hikariConfig.setTransactionIsolation(config.transactionIsolationLevel.name());
+    }
```

Comment:
So we want set the transaction isolation when creating the DB instance and it will stay that level for the DB instance's whole life? i thought we want to set the isolation level only at specific method (eg soft delete): eg
```
softDelete function: 
      set isolation level
      do soft delete
      in finally reset the isolation level to its default

```

Reply:
Yeah, true. This sets transaction isolation level for all connections to DB. I was thinking that there is no harm in setting it for all queries. I think MySql team also suggested same. It also makes debugging easier.